packages/p2p-v2/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { memo } from 'react';
+import './Avatar.scss';
+
+type TAvatarProps = {
+    name: string;
+};
+
+export const getShortNickname = (nickname: string): string => nickname?.substring(0, 2).toUpperCase();
```

Comment:
aren't we using this for online status as well? can we move this to utils?

Reply:
Noted, I can fix it in the next PR I'm working on right now, cuz I need this profile content for constructing the profile page